src/main/java/bio/terra/service/upgrade/Migrate.java

```diff
@@ -76,4 +76,16 @@ public void migrateDatabase() {
         }
     }
 
+    /**
+     * Helper function to block "drop all on start" from happening on undesired databases
+     */
+    public boolean allowDropAllOnStart() {
+        boolean allowedProfile = Arrays.stream(env.getActiveProfiles()).anyMatch(env -> env.contains("dev")
```

Comment:
I know it's not from this PR but this confusing because you're re-scoping `env` (e.g. it's a member variable for the class and you're also using it in the lambda). Maybe change it to `.anyMatch(e -> e.contains(....`